apps/datahub/src/app/home/search/search-filters/search-filters.component.html

```diff
@@ -63,6 +63,30 @@ <h2 class="text-xl mr-4 font-title" translate>
         (toggled)="toggleSpatialFilter($event)"
       ></gn-ui-check-toggle>
     </div>
+    <div class="sm:col-span-2" *ngIf="userId">
```

Comment:
I believe that this filter is supposed to be shown only in the advanced filters (cf #629 ), therefore is should have `isOpen` as a condition.

```suggestion
    <div class="sm:col-span-2" *ngIf="userId && isOpen">
```